docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,162 @@
+---
+slug: /reference/system-properties
+---
+
+# Paper System Properties 
+
+These system properties can be set when you start your server allowing for the configuration of various settings.
+
+## How they work
+
+System properties are set when you start your server. For example, if you are using a `.bat` or a `.sh` file to start your server, you can add the system properties to the file. For example:
+
+```bash
+java -Dpaper.log-level=FINE -jar paper.jar
+```
+
+Where a `-D` is used to set a system property, and the system property is `paper.log-level` with a value of `FINE`. Otherwise, just add them to the start command
+
+## List of System Properties
+
+#### paper.playerconnection.keepalive:
+
+- **default**: `30`
+- **description**: Controls how long the player connection will wait before closing in seconds.
+
+#### paper.ticklist-excessive-delay-threshold:
+
+- **default**: `60`
+- **description**: Sets the excessive tick delay to the specified number of seconds.
+
+#### timings.bypassMax:
+
+- **default**: `unset`
+- **description**: Allows for bypassing the max number of frames to send to the Aikar's Timings API. Setting this will not permit bypassing the limit unless the API is configured to allow it.
+
+#### LetMeReload:
+
+- **default**: `unset`
+- **description**: Overrides the need to send confirmation message when using the `/reload` command.
+
+#### paper.disableChannelLimit:
+
+- **default**: `unset`
+- **description**: Disables the channel limit for the server.
+
+#### net.kyori.adventure.text.warnWhenLegacyFormattingDetected:
+
+- **default**: `true`
+- **description**: Enables or disables the warning when legacy formatting is detected.
```

Comment:
This is [disabled by default](https://github.com/PaperMC/Paper/commit/ec90a78595af4cddb6e711ab71de4ee73c59ca49).

```suggestion
- **default**: `false`
- **description**: Enables or disables the warning when legacy formatting is detected in a chat component.
```